### ISSUE TITLE
Disable auto-discovery of new devices without disabling auto-discovery of existing sensors

### ISSUE THREAD

**Issue by srnoth** [custom-components/ble_monitor#685]
Currently, with "Discover devices and sensors automatically" enabled, I am able to discovery my Inkbird and Xiaomi sensors no problem. However it is also discovering my neighbor's sensors and auto-adding them to HA.

If I disable this option, neighbor's devices are not auto-added, but also none of my existing devices read from their sensors. The devices are still in HA but the sensors (temperature etc.) read as unknown. 

I looked through the documentation but there does not appear to manually define the individual sensors, nor a way to disable auto-discovery at the global level while leaving it enable at the device level.

My ideal use case would be to be able to manually add devices by MAC address, but have this integration auto-discovery their various sensors.

**Comment by Ernst79** [comment]
If they are added to the "devices" pull down menu in the BLE monitor options, it should work like you describe. If not, you have found a bug.

**Comment by srnoth** [comment]
![image](https://user-images.githubusercontent.com/69703084/152045034-477ec330-432c-4138-a243-c5be422d2136.png)

![image](https://user-images.githubusercontent.com/69703084/152045068-4dc178eb-9113-410b-b27b-ceef8fc61afa.png)

![image](https://user-images.githubusercontent.com/69703084/152045095-efcc83cd-5037-4808-a214-6f1cd9fb6caa.png)

The moment I re-enable "Discover devices and sensors automatically" I get sensor values again.

**Comment by srnoth** [comment]
Is there anything I can pull from the logs that would be helpful?

**Comment by Ernst79** [comment]
I try to reproduce your issue, but it works fine on my end. Which Xiaomi sensor do you have

**Comment by srnoth** [comment]
Sorry this appears to be only an issue with the Inkbird sensors (IBS-TH2 specifically). The Xiaomi sensors continue to pull data as expected.

**Comment by Ernst79** [comment]
found it, I think. In ble_parser/inkbird.py, we probably need to remove the .lower() in this line

```
    if self.discovery is False and source_mac.lower() not in self.sensor_whitelist:
```

**Comment by srnoth** [comment]
That worked perfectly. Manually edited ble_parser/inkbird.py on one of my installs and pulling sensor data correctly now, even with discovery off.

Note that the following parsers also have the ".lower()" in them for the whitelist check:
oral_b.py
moat.py
jinou.py

**Comment by Ernst79** [comment]
Fixed in 7.5.0
